phc-rails/spec/models/event_registration_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+describe EventRegistration, type: :model do
+  describe "#to_salesforce_object" do
+    let(:account) { Account.create(sf_id: 123) }
+    let(:event_registration) do
+      EventRegistration.create({
+          account_id: account.id,
+          Feedback__c: "great",
+          Experience__c: 5,
+          Services_Needed__c: ["kick boxing", "taichi"]
+        })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.